app/frontend/components/main-page.tsx

```diff
@@ -16,7 +16,7 @@ import { ArrowBackIcon, ArrowForwardIcon } from './arrows';
 
 export const MainPage = () => {
   const [viewType, setViewType] = React.useState<ImageView | undefined>();
-  const [singleImageViewIndex, setSingleImageViewIndex] = React.useState(0);
+  const [singleImageViewIndex, setSingleImageViewIndex] = React.useState(1);
```

Comment:
The images always come back from S3 in alphabetical order, so `base` should always come first, then `diff` second